src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double firstLeg;
+
+    public Square(String color, double firstLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+    }
+
+    public double calculateArea() {
```

Comment:
The `calculateArea()` method should be marked with `@Override` annotation since it is likely to be a method defined in an interface that this class is implementing.